### PR TITLE
chore(layout): use bp3 dropdown for lang switcher

### PR DESCRIPTION
**Pull request by EFF**
rewrite of language switcher with common dropdown.  I'm aware style isn't exactly the same but I believe it's good enough as it touches a tiny portion of the user base.

If you guys don't agree, you are welcome on styling this thing.

@sebburon should fix your issue

Closes https://github.com/botpress/botpress/issues/11405

### CODE REVIEW

**Comment by sebburon** [comment]
Just tested and it looks great and fixes [my issue ](https://github.com/botpress/botpress/issues/11405)

File: packages/studio-ui/src/web/components/Layout/StatusBar/LangSwitcher.tsx [hunk 1]
```diff
@@ -1,122 +1,65 @@
...
 
-class LangSwitcher extends React.Component<Props> {
-  elems = {}
-
-  componentDidMount() {
-    this.restoreLastLanguage()
-  }
+export const LanguageSwitcher = (props: Props) => {
+  useEffect(() => {
+    restoreLastLanguage()
```
Comment (rob-gordon) [inline packages/studio-ui/src/web/components/Layout/StatusBar/LangSwitcher.tsx:22]: I don't think you need the `restoreLastLanguage` function– you can just move the contents of the function inside of this useEffect

**Review by rob-gordon — APPROVED** [review]
Love that you converted the class component to a functional one. Small suggestion on code simplicity but not required. LGTM 👍🏻

File: packages/studio-ui/src/web/components/Layout/StatusBar/LangSwitcher.tsx (hunk 1 above)
Comment (EFF) [reply packages/studio-ui/src/web/components/Layout/StatusBar/LangSwitcher.tsx]: done ✅

**Comment by EFF** [comment]
@allardy Let's agree to drop the keyboard shortcut I'm pretty sure I'm the only one that uses the keyboard shortcut.

**Review by sebburon — APPROVED** [review]
Looking good

**Comment by allardy** [comment]
@EFF totally agree

**Pull request merged**